frontend/src/typings/vendor/msw-storybook-addon.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'msw-storybook-addon' {
```

Comment:
was this autogenerated / why is it necessary?

Reply:
There are no types for that package, so I had to create one. These particular types are retrieved from a languished PR in that repository